turbot/resource_turbot_policy_setting.go

```
@@ -221,6 +239,14 @@ func resourceTurbotPolicySettingUpdate(d *schema.ResourceData, meta interface{})
 	input := mapFromResourceData(d, getPolicySettingUpdateProperties())
 	input["id"] = id
 
+	var err error
+	if value, ok := d.GetOk("template_input"); ok {
```

ignore error, add comment